### DISCUSSION TITLE
Create a component with Q-editor

### DISCUSSION THREAD

**Discussion by mJeromeDiaz** [quasarframework/quasar#13384] [General - Components / Directives / etc] [answered]
Hi all, 
I use the q-editor component several times in many pages.
For prophylactic purposes, I would like to integrate the q-editor into a component (the parameters will be entered only once, in my component)
But I can't figure out how to return the content (v-model) from q-editor to its parent.
An idea ? thank you :)

**Comment by fairking** [comment, marked as answer]
Please read the basics of vuejs. In order to start implementing such components you must fully understand the topic: 
https://vuejs.org/guide/components/events.html#usage-with-v-model

Also v2 docs which also can be applied to v3 (in my case have a better explanation): 
https://v2.vuejs.org/v2/guide/components.html?#Using-v-model-on-Components
https://v2.vuejs.org/v2/guide/components-custom-events.html#Customizing-Component-v-model

Ones you get the point, you can literally do everything you want in vue (including one of your thousands questions in this discussion).

**Reply by mJeromeDiaz** [reply]
Thanks for your time and your link
